Tools/wpe/backends/fdo/WindowViewBackend.cpp

```diff
@@ -141,7 +141,7 @@ const struct wl_registry_listener WindowViewBackend::s_registryListener = {
         auto* window = static_cast<WindowViewBackend*>(data);
 
         if (!std::strcmp(interface, "wl_compositor"))
-            window->m_compositor = static_cast<struct wl_compositor*>(wl_registry_bind(registry, name, &wl_compositor_interface, 1));
+            window->m_compositor = static_cast<struct wl_compositor*>(wl_registry_bind(registry, name, &wl_compositor_interface, 4));
```

Comment:
Why do we need the bump to version 4 of the compositor interface? If needed to use version 4, we could use `std::max(4, interfaceVversion)` where `interfaceVersion` would be the last `uint32_t` parameter of the `.global` callback, and disable damage tracking if the chosen version is smaller than 4.